index.js

```diff
@@ -3,13 +3,14 @@ const fs = require('fs')
 const path = require('path')
 const u = require('@elife/utils')
 const exec = require('./exec')
-
+const git = require('nodegit')
```

Comment:
Good commits should be small and contain only one change. See for example: https://deepsource.io/blog/git-best-practices/